testlib/dbus.py

```diff
@@ -398,6 +399,11 @@ def fs_create(pool_path, fs_name):
             StratisDbus._POOL_IFACE,
         )
 
+        if fs_size is not None:
```

Comment:
It is usually better to just conditionalize the thing that changes, not the whole expression. Here you always call ```CreateFilesystem``` it's just when ```fs_size``` is not ```None``` the argument needs to be different. So you need a structure more like:

```
file_spec = <expression that depends on fs_size>
return iface.CreateFilesystem(
    [(fs_name, file_spec)], timeout...
)
```

Reply:
Should be all set.